addons/binding/org.openhab.binding.cm11a/src/main/java/org/openhab/binding/cm11a/handler/Cm11aAbstractHandler.java

```diff
@@ -74,14 +74,18 @@ public Cm11aAbstractHandler(Thing thing) {
 
     @Override
     public void initialize() {
+        final String NO_BRIDGE_ERROR = "No bridge found using house unit code ";
```

Comment:
There is no such concept as a method-local constant in Java, can you make it a class level constant.